boxsdk/client/client.py

```diff
@@ -159,6 +159,143 @@ def collaboration(self, collab_id):
         """
         return self.translator.translate('collaboration')(session=self._session, object_id=collab_id)
 
+    def legal_hold_policy(self, policy_id):
+        """
+        Initialize a :class:`LegalHoldPolicy` object, whose box id is policy_id.
+
+        :param policy_id:
+            The box ID of the :class:`LegalHoldPolicy` object.
+        :type policy_id:
+            `unicode`
+        :return:
+            A :class:`LegalHoldPolicy` object with the given entry ID.
+        :rtype:
+            :class:`LegalHoldPolicy`
+        """
+        return self.translator.translate('legal_hold_policy')(session=self._session, object_id=policy_id)
+
+    def legal_hold_policy_assignment(self, policy_assignment_id):
+        """
+        Initialize a :class:`LegalHoldPolicyAssignment` object, whose box id is policy_assignment_id.
+
+        :param policy_assignment_id:
+            The assignment ID of the :class:`LegalHoldPolicyAssignment` object.
+        :type policy_assignment_id:
+            `unicode`
+        :return:
+            A :class:`LegalHoldPolicyAssignment` object with the given entry ID.
+        :rtype:
+            :class:`LegalHoldPolicyAssignment`
+        """
+        return self.translator.translate('legal_hold_policy_assignment')(session=self._session, object_id=policy_assignment_id)
+
+    def legal_hold(self, policy_id):
+        """
+        Initialize a :class:`LegalHold` object, whose box id is policy_id.
+
+        :param policy_id:
+            The assignment ID of the :class:`LegalHold` object.
+        :type policy_id:
+            `unicode`
+        :return:
+            A :class:`LegalHold` object with the given entry ID.
+        :rtype:
+            :class:`LegalHold`
+        """
+        return self.translator.translate('legal_hold')(session=self._session, object_id=policy_id)
+
+    def create_legal_hold_policy(
+            self,
+            policy_name,
+            description=None,
+            filter_starting_at=None,
+            filter_ending_at=None,
+            is_ongoing=None
+    ):
+        """
+        Create a legal hold policy.
+        :param policy_name:
```

Comment:
I think you might need a blank line between the method description and the first param description here